src/components/ActionList/ActionList.scss

```diff
@@ -25,6 +22,10 @@ $item-vertical-padding: ($item-min-height - line-height(body)) / 2;
   margin: 0;
   padding: 0;
   border-top: border();
+
+  &.newDesignLanguage {
+    padding: spacing(tight) spacing(tight) 0 spacing(tight);
```

Comment:
Not sure we should have the top padding here
![image](https://user-images.githubusercontent.com/1972567/94014148-27462500-fd79-11ea-887d-8c7e4a5cee84.png)

Reply:
This should be addressed in the latest commit.
![Screen Shot 2020-09-23 at 11 00 33 AM](https://user-images.githubusercontent.com/19199063/94030675-194dcf80-fd8c-11ea-9a22-d1dc120c01a4.png)
![Screen Shot 2020-09-23 at 11 00 29 AM](https://user-images.githubusercontent.com/19199063/94030679-19e66600-fd8c-11ea-96df-69ca80f614c2.png)